ecosystem.config.yml

```diff
@@ -0,0 +1,7 @@
+apps:
+  - script: ./src/index.js
+    name: 'memebot-api'
+    env:
```

Comment:
Still need the MEMEBOT_IP env var as well?

Reply:
Its [Node Version Manager](https://github.com/nvm-sh/nvm), it lets you install multiple versions of node on the user. Basically we can update our node version just by updating `.nvmrc`.

Reply:
Yeah should probably add that. I dunno if the way we verify that is the safest, should probably setup some kind of rsa key or something?

Reply:
Agreed, but if you are planning on dumping the config, then we need something that works for now.